### PR TITLE
feat(chrome extension): add transact component

### PR DESCRIPTION
**Pull request by groundedSAGE**

### CODE REVIEW

**Comment by groundedSAGE** [comment]
The namespace `datalog-console.workspaces.transact-cards` is currently not functional. I am thinking of either two options:

1. We keep the namespace and just add a note into it that we would like to have a card for it
2. We remove the namespace/card entirely

File: src/main/datalog_console/client.cljs [hunk 1]
```diff
@@ -30,15 +30,24 @@
...
     (.addListener (gobj/get port "onMessage")
                   (fn [msg]
-                    (when-let [db-str (gobj/getValueByKeys msg ":datalog-console.remote/remote-message")]
-                      (reset! rconn (d/conn-from-db (cljs.reader/read-string db-str))))))
+                    (when-let [remote-message (cljs.reader/read-string (gobj/getValueByKeys msg ":datalog-console.remote/remote-message"))]
+                      (cond
+                        (d/db? remote-message)
+                        (reset! rconn (d/conn-from-db remote-message))
+
+                        (= :success remote-message)
```
Comment (groundedSAGE) [inline src/main/datalog_console/client.cljs:38]: This still has to change. Need to think of a better pattern for handling different types of results when interacting with remote databases. Perhaps adopting HTTP status responses is the best approach?

File: src/main/datalog_console/integrations/datascript.cljs [hunk 2]
```diff
@@ -1,6 +1,14 @@
 (ns datalog-console.integrations.datascript
   (:require [goog.object :as gobj]
-            [cljs.reader]))
+            [cljs.reader]
+            [datascript.core :as d]))
+
+(defn transact-from-remote! [conn transact-str]
+  (try
+    (d/transact conn (cljs.reader/read-string transact-str))
+    :success
```
Comment (groundedSAGE) [inline src/main/datalog_console/integrations/datascript.cljs:9]: Again maybe a better pattern will help here.

File: src/main/datalog_console/components/transact.cljs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+         [:textarea
+          {:style {:min-width "20rem"}
+           :class        "border p-2"
+           :rows          5
+           :value        @transact-text
+           :on-change    (fn [e]
+                           (reset! transact-text (goog.object/getValueByKeys e #js ["target" "value"])))}]
+         [:button {:type "submit"
+                   :class "py-1 px-2 rounded-b bg-gray-200 border"}
+          "Run transaction"]]]
```
Comment (smothers) [inline src/main/datalog_console/components/transact.cljs:25]: maybe just "Transact"

File: src/main/datalog_console/lib/version.cljc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+(ns datalog-console.lib.version
+  #?(:cljs (:require-macros [datalog-console.lib.version :refer [fetch-version]]))
+  #?(:clj (:require [clojure.data.xml :as xml])))
+
+#?(:clj (defmacro fetch-version []
```
Comment (smothers) [inline src/main/datalog_console/lib/version.cljc:5]: neat!

**Review by smothers — APPROVED** [review]

**Pull request merged**